tensorflow/core/protobuf/config.proto

```diff
@@ -257,7 +273,7 @@ message GPUOptions {
   // Everything inside experimental is subject to change and is not subject
   // to API stability guarantees in
   // https://www.tensorflow.org/guide/version_compat.
-  Experimental experimental = 9;
+  Experimental experimental = 10;
```

Comment:
Let's keep the tag number unchanged. 
it's fine the tag numbers are out of order